tpm2/structures.go

```diff
@@ -888,7 +888,8 @@ type Name struct {
 	Digest *HashValue
 }
 
-func decodeName(in *bytes.Buffer) (*Name, error) {
+// DecodeName deserializes a Name hash from the TPM wire format.
```

Comment:
Maybe mention this is a `TPM2B_NAME` structure explicitly?

Reply:
Done,  added a comment on the struct itself.